CPAC/pipeline/schema.py

```diff
@@ -985,6 +991,8 @@ def sanitize(filename):
     },
     'amplitude_low_frequency_fluctuation': {
         'run': bool,
+        #'target_space': In({'Template', 'Native'}),
```

Comment:
You can remove this comment and https://github.com/FCP-INDI/C-PAC/blob/e81170fffbedfd11a1c04c0f402a1ad052e923c2/CPAC/pipeline/schema.py#L1012

---

I noticed these options are in a set in the commented out code and in a list in the reusable options at the top. I think either structure would be fine (and that tuples might actually be most efficient the way to go?). This comment is mostly for me, I think. I'll probably open an issue to make the schema more efficient by changing some of these structures to tuples in the future.